### PR TITLE
Minor Clean up

### PR DESCRIPTION
**Pull request by jdconrad**
- Minor clean up of Writer constants.
- Fixed Writer imports.
- Removed synthetic attribute from the generated constructor and method.
- Added a safeguard for maximum script length.

### CODE REVIEW

File: plugins/lang-painless/src/main/java/org/elasticsearch/painless/Compiler.java [hunk 1]
```diff
@@ -38,6 +38,11 @@
  */
 final class Compiler {
     /**
+     * The maximum number of characters allowed in the script source.
+     */
+    protected static int MAXIMUM_SOURCE_LENGTH = 16384;
```
Comment (rjernst) [inline plugins/lang-painless/src/main/java/org/elasticsearch/painless/Compiler.java]: pkg private final?

**Comment by rjernst** [comment]
LGTM

File: plugins/lang-painless/src/main/java/org/elasticsearch/painless/Compiler.java (hunk 1 above)
Comment (jdconrad) [inline plugins/lang-painless/src/main/java/org/elasticsearch/painless/Compiler.java]: Oops, yeah.

**Comment by jdconrad** [comment]
Thanks @rjernst

**Pull request closed without merging**